modules/data/log/tmpl/index.tmpl

```diff
@@ -0,0 +1,40 @@
+<? I18N znc-log ?>
+<? INC Header.tmpl ?>
+
+<? IF Files ?>
+<h3>File Browser</h3>
+<div class="sectionbg">
+<div class="sectionbody">
+<ul style="list-style: none;">
+<? LOOP Files ?>
+<li><a href="<? VAR ModPath TOP ?>?path=<? VAR Long ESC=HTML,URL ?>"><? VAR Short ?></a></li>
+<? ENDLOOP ?>
+</ul>
+</div>
+</div>
+<? ENDIF ?>
+
+<? IF Log ?>
+<h3>Log Viewer (Page <? VAR Page ?>)</h3>
+<div class="sectionbg">
+<div class="sectionbody">
+<code style="display: block; margin-bottom: 0.5em;">
+<? LOOP Log ?>
+<? VAR Line ?><br>
+<? ENDLOOP ?>
+</code>
+<? IF PrevPage || PrevPage == 0 ?>
+<button><a style="text-decoration: none;" href="<? VAR ModPath TOP ?>?path=<? VAR Path ESC=HTML,URL ?>&page=<? VAR PrevPage ESC=HTML,URL ?>">Prev</a></button>
```

Comment:
Prev/Next should be translateable. In .tmpl files it can be done via `<? FORMAT "Foo"  ?>`

Same for other strings

Reply:
Ok. I'll see about doing that once I know what I will be revising in the template and module code.

Reply:
All user-visible text should now be translateable in the template save for ones where no translation is feasible.